masonry/src/contexts.rs

```diff
@@ -309,7 +309,7 @@ impl_context_method!(
         ///
         /// [`set_disabled`]: EventCtx::set_disabled
         pub fn is_disabled(&self) -> bool {
-            self.widget_state.is_disabled()
+            self.widget_state.is_disabled
```

Comment:
I mean that's one of the nittiest nits: Why this change (and at the other places too)?
I feel like an abstraction i.e. function call potentially allows in the future to wrap more stuff in it, while this would need to be refactored again to such a function call when that would be the case, and is otherwise optimized anyways.

Reply:
I try to avoid function calls when not necessary. Even if they're inlined, there's still a cognitive penalty of not knowing how much / which code the function executes.

Someone trying to understand this code and reading the function version needs (1) to jump to the `is_disabled()` function definition, then (2) look for code setting the value of the `is_disabled` field. If you replace the function with its field access, that someone only needs to do part 2.